src/main/java/com/strandls/authentication_utility/util/AuthUtil.java

```diff
@@ -122,6 +125,41 @@ public static void updateUserProfile(CommonProfile profile, Long userId, String
 		return response;
 	}
 
+	public static Map<String, Object> generateAccessTokenFromEmail(String email) {
+		Map<String, Object> response = new HashMap<>();
+		if (!email.isEmpty() && email != null) {
```

Comment:
Shouldn't we first check for a null string and then for an emtpy string? otherwise if the string is null,it will generate an exception while checking for email.isEmpty() first.
Can you please have a look?